.github/ISSUE_TEMPLATE/pre_release_checklist.md

```diff
@@ -12,23 +12,23 @@ Prior to moving into production, the following steps should be completed or conf
 - [ ] QA to schedule the release with staff/clients (daytime's best or when staff are available for rollback)
 - [ ] All dev/test pipeline test suites green
 - [ ] Dev/QA chat to plan prod verification testing (unless already automated)
-- [ ] Release the code to production and complete smoke test
+- [ ] Release the code to production and complete smoke test (STEPS BELOW)
 - [ ] Finalise/publish the release in GitHub, tagging it
+- [ ] Merge release branch back to master (if applicable)
+- [ ] Change openshift builds/pipelines back to master (if applicable)
 
+**Deployment Steps**
 
-**NameX Smoke Test Script**
-- [ ] Log in
-- [ ] Go to search
-- [ ] Filter for last 30 days
-- [ ] Go to ZZZ record (usually on Hold)
-- [ ] edit name 1, click save
-- [ ] edit address line 1 + comment, click save
-- [ ] check NRO for changes
-- [ ] reject name 1 with a conflict
-- [ ] approve name 2 with quick approve
-- [ ] wait 2 minutes, then check in NRO
-- [ ] reload NR in namex
-- [ ] reset NR
-- [ ] undo all decisions
-- [ ] change name 1 and address line 1 back to original values
+Dev:
+- [ ] config map changes
+  - list here, copy to test and prod
+

```

Comment:
Does it make sense to add some of the other items from past tickets, such as:
- dependencies (e.g., sbc-auth needs to be deployed first, or keycloak config needs to be updated)
- data migration changes
?

Reply:
Yes, I'd say so. I'd elaborate on data migration, to say are there any extra one-time scripts that need to be run. I'll add that and the auth/keycloak note.

Reply:
Updated, please re-review.